doc/Learn/Tutorials/Markup/HowTo-CustomMarkupProject-Theme.md

```diff
@@ -0,0 +1,315 @@
+##### For a test, change the ColorPaletteOverride and run the project.
+
+We change the Theme.Colors.Background.Default.Key and will be able to check the change if we customize the Background on the Page.
+
+```csharp
+//From
+//.Add<Color>(Theme.Colors.Background.Default.Key, "#F4F4F4", "#000000")
+//To
+.Add<Color>(Theme.Colors.Background.Default.Key, "#A1B2C3", "#000000")
+```
+
+For use the color from the theme open the MainPage.cs and change the background of the page.
+
+```csharp
+public MainPage()
+	{
+		this
+            //Remove the ApplicationPageBackgroundThemeBrush Background
+            //.Background(ThemeResource.Get<Brush>("ApplicationPageBackgroundThemeBrush"))
+
+            //Use the Direct the Uno.Themes.Markup.Theme.Brushes
+            //.Background(Theme.Brushes.Background.Default)
+
+            //Or use the ResourceKeyDefinition to have the same result
+            .Background(ThemeResource.Get<Brush>("BackgroundBrush"))
+```
+
+You always can back on the Material documentation and learning how the [Baseline color scheme](https://m3.material.io/styles/color/the-color-system/tokens) works
+
+
+## Cupertino
+
+We do not suport Cupertino at the moment.
+
+## Quick Start
+
+We are able to change from Dark Mode and Light Mode direct on the application.
+So we can use the ColorPaletteOverride as both, Dark and Light Mode.
+
+To do this, we can create a simple control toggle button and switch between modes.
+
+- In the Shared Project open the file *MainPage.cs* and change the content to have the ToggleButton.
+
+```csharp
+new Grid().RowDefinitions<Grid>("*, *")
+	.Children(
+		new NavigationBar()
+			.Grid(column: 0)
+			.HorizontalAlignment(HorizontalAlignment.Left)
+			.Content("Title Main Page"),
+		new ToggleButton()
+			.Grid(column: 0)
+			.Content("Toggle Theme")
+			.HorizontalAlignment(HorizontalAlignment.Right)
+			.Assign(out var toggle)
+	),
+```
+
+- After that we need to add a Click EventHandler to Toggle the Theme. 
+
+```csharp
+toggle.Checked += (s, e) =>
+{
+	if (Application.Current is App app)
+	{
+		app.AppTheme?.SetTheme(true);
+	}
+};
+toggle.Unchecked += (s, e) =>
+{
+	if (Application.Current is App app)
+	{
+		app.AppTheme?.SetTheme(false);
+	}
+};
+```
+
+- But to change the theme, we need to have access to the Window.
+- So we will create a new Class Named AppTheme.cs and add this content.
+- And doing this, we can get benefit from the Toolkit helper [SystemThemeHelper](https://platform.uno/docs/articles/features/working-with-themes.html?tabs=windows).
+
+```csharp
+namespace MySampleToolkitProject;
+public class AppTheme
+{
+	private readonly Window _window;
+	public AppTheme(Window window)
+	{
+		_window = window;
+	}
+
+    public void SetTheme(bool darkMode)
+	{
+			SystemThemeHelper.SetRootTheme(_window.Content.XamlRoot, darkMode);
+	}
+}
+
+```
+And for this pedagogical purpose, we can update App.cs and include a public class AppTheme so that it can be used in our control, to update the theme.
```

Comment:
```suggestion
And for this educational purpose, we can update `App.cs` and include a public class `AppTheme` so that it can be used in our control, to update the theme.
```